### PR TITLE
Chapter3: Mostly static pages

### PR DESCRIPTION
**Pull request by gdhaison**

### CODE REVIEW

**Comment by gdhaison** [comment]
ready

File: app/views/static_pages/about.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<h1>About</h1>
+<p>
+  The <a href="http://www.railstutorial.org/"><em>Ruby on Rails
+  Tutorial</em></a> is a
+  <a href="http://www.railstutorial.org/book">book</a> and
+  <a href="http://screencasts.railstutorial.org/">screencast series</a>
+  to teach web development with
+  <a href="http://rubyonrails.org/">Ruby on Rails</a>.
+  This is the sample application for the tutorial.
+</p>
```
Comment (thanhhungchu95) [inline app/views/static_pages/about.html.erb]: add an empty line below

File: app/views/static_pages/help.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<h1>Help</h1>
+<p>
+  Get help on the Ruby on Rails Tutorial at the
+  <a href="http://www.railstutorial.org/help">Rails Tutorial help page</a>.
+  To get help on this sample app, see the
+  <a href="http://www.railstutorial.org/book"><em>Ruby on Rails Tutorial</em>
+  book</a>.
+</p>
```
Comment (thanhhungchu95) [inline app/views/static_pages/help.html.erb]: add an empty line below

File: app/views/static_pages/home.html.erb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<h1>Sample App</h1>
+<p>
+  This is the home page for the
+  <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+  sample application.
+</p>
```
Comment (thanhhungchu95) [inline app/views/static_pages/home.html.erb]: add an empty line below

File: config/routes.rb [hunk 4]
```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
```
Comment (thanhhungchu95) [inline config/routes.rb]: remove this line

File: config/routes.rb [hunk 5]
```diff
@@ -1,3 +1,7 @@
...
+  get  'static_pages/home'
```
Comment (thanhhungchu95) [inline config/routes.rb]: in file *.rb, use double-quote instead of single-quote

File: app/views/static_pages/about.html.erb [hunk 6]
```diff
@@ -0,0 +1,10 @@
+<h1>About</h1>
+<p>
+  The <a href="http://www.railstutorial.org/"><em>Ruby on Rails
```
Comment (thanhhungchu95) [inline app/views/static_pages/about.html.erb]: use `link_to` helper instead of `<a>` tag

File: app/views/static_pages/about.html.erb [hunk 7]
```diff
@@ -0,0 +1,10 @@
...
+  Tutorial</em></a> is a
+  <a href="http://www.railstutorial.org/book">book</a> and
+  <a href="http://screencasts.railstutorial.org/">screencast series</a>
+  to teach web development with
```
Comment (thanhhungchu95) [inline app/views/static_pages/about.html.erb]: research about i18n and apply with the plain text

**Comment by gdhaison** [comment]
ready

**Comment by gdhaison** [comment]
ready

**Comment by gdhaison** [comment]
ready

**Comment by gdhaison** [comment]
ready

**Comment by gdhaison** [comment]
hinh ch p t 2018-12-25 12-58-41

**Comment by gdhaison** [comment]
ready

File: config/routes.rb [hunk 8]
```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get "static_pages/home"
+  get "static_pages/help"
+  get "static_pages/about"
+  root "static_pages#home"
```
Comment (DuongNT1512) [inline config/routes.rb]: khai báo 2 resource trỏ tới home hả em?>

**Comment by gdhaison** [comment]
oke chị, e cảm ơn ạ

**Pull request merged**